### DISCUSSION TITLE
Tests no longer need to be private?

### DISCUSSION THREAD

**Discussion by remy90** [xunit/xunit#2317] [Question] [answered]
I can't find any information on this online, as most searches refer to BL private methods. Do test methods no longer need to be public? The examples in your docs all show them to be public but it seems to be working for me locally.

i.e.

```
[Fact]
private void test()
{
    Assert.True(true);
} // Runs and passes
```

**Comment by bartelink** [comment, marked as answer]
The methods never needed to be public, even in V1, but the classes do need to be ([perf for reflection against non-public Types is worse](https://stackoverflow.com/a/16214704/11635))
